sigma/plugins/utility/other/define.py

```diff
@@ -0,0 +1,94 @@
+import discord
+import aiohttp
+from bs4 import BeautifulSoup
+
+async def define(cmd, message, args):
+
+    definition = ""
+    defNum = 1
+    defLast = False
+    hasPrinted = False
+    defURL = r"https://www.merriam-webster.com/dictionary/"
+
+    if not args:
+        await message.channel.send(cmd.help())
+        return
+    try:
+        defNum = int(args[-1])
+        defLast = True
+    except ValueError:
+        pass
+
+    if not defLast:
+        for i in args:
+            defURL += i
+            defURL += "%20"
+    else:
+        argsCopy = args.copy()
+        del(argsCopy[-1])
+        for i in argsCopy:
+            defURL += i
+            defURL += "%20"
+
+    async with aiohttp.ClientSession() as session:
+        async with session.get(defURL) as resp:
+            rawDef = await resp.text()
+    
+    try:
+        rawDef = rawDef[rawDef.index("<!--JC: actual definitions-->")-10:]
+    except:
+        embed = discord.Embed(title=":exclamation: The information you entered does not correlate to a Dictionary Entry.", color=0xFF0000)
+        await message.channel.send(None, embed=embed)
+        return
+
+    rawDefArr = rawDef.split("<!--JC: actual definitions-->")
+
+    soupObj = BeautifulSoup(rawDefArr[defNum], "html.parser")
+
+    try:
+        for tag in soupObj.find_all(class_="intro-colon"):
+            definition += tag.text
+
+            if not definition:
+                raise IndexError("The website has changed, please fix.")
+
+            if(len(tag.next_sibling.strip()) == 0):
+                if hasPrinted:
+                    continue
+                for tag2  in soupObj.find_all(class_="sx-link sc"):
+
+                    defCopy = definition
+
+                    definition += tag2.text
+
+                    if defCopy is definition:
+                        raise IndexError("The website has changed, please fix.")
+
+                    definition += " : "
+
+                hasPrinted = True
+            else:
+                definition += tag.next_sibling
+                definition += " "
+    except:
+        cmd.log("The website has changed; either \"intro-colon\" or \"sx-link sc\" are not present.")
+        return
+
+    definition.strip()
+    definition.replace(":"," \n— ")
+
+    if "The word you've entered" in definition:
+        embed = discord.Embed(title=":smile: That definition doesn't exist, silly!", color=0xFF8EE6)
+        await message.channel.send(None, embed=embed)
+        return
+    if "Sorry, the word you’re looking for" in definition:
```

Comment:
I add spaces between ends and beginnings of multi-line conditional statements for readability